packages/hub/tasks/notify-customer-payment.ts

```diff
@@ -65,9 +69,23 @@ export default class NotifyCustomerPayment {
       );
     }
 
+    let merchantName = '';
+
+    try {
+      if (result.merchantSafe?.infoDid) {
+        let merchantInfo = await this.merchantInfo.getMerchantInfo(result.merchantSafe.infoDid);
+
+        if (merchantInfo) {
+          merchantName = ` ${merchantInfo.name}`;
+        }
+      }
+    } catch (e) {
+      // Silently ignore and just show notification without merchant name
+    }
+
     let notifiedAddress = result.merchant.id;
     let spendAmount = result.spendAmount;
-    let message = `Your business received a payment of §${spendAmount}`;
+    let message = `Your business${merchantName} received a payment of §${spendAmount}`;
```

Comment:
Will also ask about this v `[Merchant name] received a new payment.` in CS-2616.

Reply:
Thanks for checking in, I was not thinking too much about wording pending concrete answers, but I’ve updated in c281ddd80 with the decision there.